### PR TITLE
Automated Test - COMMERCE-6102 | master

### PR DESCRIPTION
**Pull request by PauloVAS**

### CODE REVIEW

**Comment by PauloVAS** [comment]
https://issues.liferay.com/browse/COMMERCE-6102, CC: @PietroBocale

**Comment by pedro-michel** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceChannels.testcase [hunk 1]
```diff
@@ -146,7 +146,68 @@ definition {
...
+			value1 = "Gross Price");
+
+		Select(
+			key_title = "Discounts Target Price Type",
+			locator1 = "CommerceEntry#CATALOGS_FORM",
+			value1 = "Gross Price");
+
+		Button.clickSave();
+
+		AssertElementPresent(locator1 = "Message#SUCCESS");
```
Comment (Pietro69) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceChannels.testcase:210]: The part where it is checked if all changes have been saved is missing we have only the check if the green alert is present, the step 15 of the ticket says: "Verify if the green alert appears and if all the changed are correctly saved"

**Review by Pietro69 — CHANGES_REQUESTED** [review]

**Comment by PauloVAS** [comment]
@Pietro69 fixed

**Comment by pedro-michel** [comment]
ci:test:sf

**Review by pietrobocale — APPROVED** [review]

**Comment by pietrobocale** [comment]
ci:test:commerce-functional

**Comment by iovaneale** [comment]
ci:test:commerce-functional

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by kristinonias** [comment]
ci:test:commerce-functional

**Comment by kristinonias** [comment]
ci:forward

**Pull request closed without merging**